doc/specs/polls.md

```diff
@@ -0,0 +1,35 @@
+The new system has two poll types, both allow for multiple choices to be made and the second systems allows for choices to be weighed and ranked.
+
+## 1. Multiple Choice Polls (basic)
+* Checkboxes would appear next to each entry. A user could select multiple checkboxes.
+* The poll creator could enter a number into a "Choices Allowed" field. This value would determine the max number of choices a user could select. Leaving the field blank or entering "0" would mean unlimited choices.
+* The same goes for "Choices Required" field, except it's the minimum choices. Leaving the field blank, or entering "0" or "1" would specify that 1 choice is required.
+* If a user submits a vote total over the maximum amount, a notification will tell him that his vote wasn’t counted and that he should try again. 
+  * Alternatively: voting over the maximum amount would be impossible because the checkboxes would grey out once the maximum number of choices was met. 
+  * Alternatively # 2: voting over the maximum amount would be impossible because the submit button would disappear if too many choices were selected.
```

Comment:
Agreed. 

Also we shouldn't explicitly mention UI in the specs unless it's necessary to how whatever you are describing functions.